tests/system_health/test_system_health.py

```diff
@@ -100,129 +101,108 @@ def test_device_checker(duthosts, rand_one_dut_hostname, device_mocker_factory):
-        mock_result, fan_name = device_mocker.mock_fan_speed(True)
-        if mock_result:
-            logging.info('Mocked valid fan speed for {}, waiting {} seconds for it to take effect'.format(fan_name,
-                                                                                                          THERMAL_CHECK_INTERVAL))
-            time.sleep(THERMAL_CHECK_INTERVAL)
-            value = redis_get_field_value(duthost, STATE_DB, HEALTH_TABLE_NAME, fan_name)
-            assert not value or expect_value not in value, 'Mock fan valid speed, expect {}, ' \
-                                                           'but it still report invalid speed'
+        fan_mock_result, fan_name = device_mocker.mock_fan_speed(False)
+        fan_expect_value = EXPECT_FAN_INVALID_SPEED.format(fan_name)
 
-        mock_result, fan_name = device_mocker.mock_fan_presence(False)
-        expect_value = EXPECT_FAN_MISSING.format(fan_name)
-        if mock_result:
-            logging.info('Mocked fan absence {}, waiting {} seconds for it to take effect'.format(fan_name,
-                                                                                                  THERMAL_CHECK_INTERVAL))
-            time.sleep(THERMAL_CHECK_INTERVAL)
-            value = redis_get_field_value(duthost, STATE_DB, HEALTH_TABLE_NAME, fan_name)
-            assert value and value == expect_value, 'Mock fan absence, expect {}, but got {}'.format(expect_value,
-                                                                                                     value)
+        asic_mock_result = device_mocker.mock_asic_temperature(False)
+        asic_expect_value = EXPECT_ASIC_HOT
 
-        mock_result, fan_name = device_mocker.mock_fan_presence(True)
-        if mock_result:
-            logging.info('Mocked fan presence for {}, waiting {} seconds for it to take effect'.format(fan_name,
-                                                                                                       THERMAL_CHECK_INTERVAL))
-            time.sleep(THERMAL_CHECK_INTERVAL)
-            value = redis_get_field_value(duthost, STATE_DB, HEALTH_TABLE_NAME, fan_name)
-            assert not value or value != expect_value, 'Mock fan presence, but it still report absence'
+        psu_mock_result, psu_name = device_mocker.mock_psu_presence(False)
+        psu_expect_value = EXPECT_PSU_MISSING.format(psu_name)
 
-        mock_result, fan_name = device_mocker.mock_fan_status(False)
-        expect_value = EXPECT_FAN_BROKEN.format(fan_name)
-        if mock_result:
-            logging.info('Mocked fan broken for {}, waiting {} seconds for it to take effect'.format(fan_name,
-                                                                                                     THERMAL_CHECK_INTERVAL))
+        if fan_mock_result and asic_mock_result and psu_mock_result:
+            logging.info('Mocked invalid fan speed for {}'.format(fan_name))
```

Comment:
Can you update the code to create a module specific logger instead of using the root logger?

Reply:
Sure

Reply:
Done